packages/ui/Scrollable/Scrollable.js

```diff
@@ -1243,6 +1250,12 @@ class ScrollableBase extends Component {
 		if (this.props.addEventListeners) {
 			this.props.addEventListeners(childRefCurrent.containerRef);
 		}
+
+		if (this.childRefCurrent.cc) { // For VirtualList
```

Comment:
Is there any better way to distinguish the type of child component rather than checking `cc`?

Reply:
While we discussed this issue, we found that Scroller also need to be resizable to show or to hide scrollbars. I'll remove the condition.